includes/definitions/fs.yaml

```diff
@@ -0,0 +1,18 @@
+os: fs
+text: 'Fiberstore'
+type: network
+icon: fs
+group: fs
+over:
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+mib_dir:
+    - fs
+discovery:
+    - sysObjectID: 
+        - .1.3.6.1.4.1.51134
```

Comment:
That is a lot of sysObjectIDs.  I'm concerned this OS isn't the right scope.  OS should be defined per OS/MIB set, not per vendor.

Reply:
OK. I don't really understand the mess they did with so many sysObjectIDs. For the moment we have 2 mib groups (GBN* and SWITCH) so we will start with 2 OSes grouped in "fs".

Reply:
I'm pretty sure all of the Fiberstore equipment is just rebranded oem equipment.

Reply:
That would make sense ... Anyway, I'll still do that split ASAP.

Reply:
Split done. Right now, 2 groups.